### DISCUSSION TITLE
Add Keyboard Shortcuts to right-click context menu

### DISCUSSION THREAD

**Discussion by spambas** [f1multiviewer/issue-tracker#214] [Feature requests]
I could not find the Keyboard short cuts for audio panning. It was not in the docs. Discord told me it was `p`.

Would like to see these shortcuts added to the context menu, or for panning even better, add to the audio options at the bottom.

![image](https://user-images.githubusercontent.com/10828880/189652550-af8ae4b7-c237-41fb-b82d-bc7204004173.png)

**Comment by JJWatMyself** [comment]
I agree that to add clickable equivalents for keyboard shortcuts would be helpful. Not just for noobs.  Examples are:

Pan Audio (p) --> sub menu with three options
Toggle overlays (t) --> sub menu with three options
Show Debug (d)  --> check mark

Right click would be a good place to have these options.  Helps folks learn the app, while using the app.

**Comment by jeppeolesen** [comment]
For reference the keyboard shortcuts can be found here: https://multiviewer.dev/docs/usage/keyboard-shortcuts
